tests/integration/mci/test_clear_value.py

```diff
@@ -44,7 +44,7 @@ def test_clear_value(self):
 
         # We submit the form
         self.post(form_data)
-        self.assertInPage(error_messages['INVALID_DATE_RANGE'])
+        self.assertInPage("Enter a 'period to' date later than the 'period from' date.")
```

Comment:
Why?

Reply:
was not passing previously

Reply:
Tests currently work in master don't they?

Reply:
Yeah they could be reverted now, i think it was the if not fields validare thing but as thats been reverted

Reply:
done